static/scss/answers/interactive-map/InteractiveMap.scss

```diff
@@ -563,6 +566,10 @@
     background-color: var(--hh-color-gray-2);
   }
 
+  &.CollapsibleFilters .yxt-VerticalResultsCount {
```

Comment:
Is this only a contrast problem when CollapsibleFilters are available? I know we add CollapsibleFilters automatically (regardless of if you use it or not) but it may be confusing to make it this CSS selector if this is a color change that should happen agnostic to collapsible filters existing

Reply:
it's just due to the lighter grey background rather than the white that the theme uses by default, so i think I can remove the cfilters selector from here